website/src/components/footer.js

```diff
@@ -54,6 +55,10 @@ const BrandingContainer = styled.div`
   ${media.palm`
     margin-top: ${props => props.theme.margins.small};
   `};
+  
+  > * {
```

Comment:
this seems overly aggressive, make everything has a margin-left: small?

Reply:
I decided to go that way in order to enforce a minimum left margin to all elements within that specific row. Before only one of the 3 elements had margin-left property set correctly.
We can definitely have a class to set the margin correctly to every single elements or we could use display which provides a way to handle space between items easily and in a more elegant way